yarn.lock

```diff
@@ -7120,7 +7120,7 @@ modularscale@^1.0.2:
   dependencies:
     lodash.isnumber "^3.0.0"
 
-moment@^2.21.0:
+moment@^2.21.0, moment@^2.24.0:
```

Comment:
Huh, didn't notice that it installed a different version of `moment` to the one used by gatsby`. 😐